amd_openvx/openvx/ago/ago_kernel_api.cpp

```diff
@@ -2215,9 +2215,8 @@ int ovxKernel_NonLinearFilter(AgoNode * node, AgoKernelCommand cmd)
 			return VX_ERROR_INVALID_FORMAT;
 		else if (!width || !height)
 			return VX_ERROR_INVALID_DIMENSION;
-		else if (node->paramList[3]->u.img.format != format) {
+		else if (node->paramList[3]->u.img.format != format)
```

Comment:
keep the brackets constant across

Reply:
@hansely can you look at this?

Reply:
They are actually constant across the nodes.

Reply:
Fixed the previous one by removing bracket.